src/lib.rs

```diff
@@ -71,10 +66,15 @@ impl BitVector {
         };
         test_value > 0
     }
+
+    /// Determine the approximate amount of memory used by this BitVector.
+    /// See the caveat in Cascade::approximate_size.
+    fn approximate_size(&self) -> usize {
+        self.bytes.len() + size_of::<usize>()
```

Comment:
nit: either the `usize` is negligible and we don't care about it, or we also care about the Vec struct and padding. I would either go with `self.bytes.len()` or `size_of::<BitVector>() + self.bytes.len()`, and I guess I lean towards the latter.